tests/integration/StorageWriter/claimData.ts

```diff
@@ -0,0 +1,31 @@
+export const workingClaim = {
```

Comment:
Could the owner of this content claim copyright in any way?
Can we obfuscate it in any way in case the owner doesn't agree with this being in an open source code repo?

Reply:
Good point @warrenv, I will see if I can find the specific issue with the content and adjust to fake data.